src/Microsoft.DocAsCode.MarkdownLite/JsonTokenTreeRenderer.cs

```diff
@@ -273,6 +278,15 @@ private string GetSize()
             return "1";
         }
 
+        private string escape(string content)
+        {
+            return content.Replace("\n", "\\n")
+                .Replace("\"", "&quot;")
+                .Replace("<", "&lt;")
+                .Replace(">", "&gt;")
+                .Replace("'", "&#39;");
```

Comment:
sometime the content has been escaped but some time not , for that if I use HtmlEncode "&gt"(>) will be translate to "&amp;gt" 

---
In reply to: [75989504](https://github.com/dotnet/docfx/pull/631#discussion_r75989504) [](ancestors = 75989504,75989168,75988652)